tests/test_reports/cnh_male_report.csv

```diff
@@ -0,0 +1,3 @@
+,ground_truth,hypothesis,audio_length,inference_end
```

Comment:
Are these results from a particular test run? They should either be removed or put in some dated folder if they are kept for later comparison